recipe/thirdpartypasswordless/emaildelivery/backwardCompatibilityService/main.go

```diff
@@ -0,0 +1,33 @@
+package backwardCompatibilityService
+
+import (
+	"errors"
+
+	"github.com/supertokens/supertokens-golang/ingredients/emaildelivery"
+	emailVerificationBackwardsCompatibilityService "github.com/supertokens/supertokens-golang/recipe/emailverification/emaildelivery/backwardCompatibilityService"
+	"github.com/supertokens/supertokens-golang/recipe/emailverification/evmodels"
+	passwordlessBackwardsCompatibilityService "github.com/supertokens/supertokens-golang/recipe/passwordless/emaildelivery/backwardCompatibilityService"
+	"github.com/supertokens/supertokens-golang/recipe/thirdpartypasswordless/tplmodels"
+	"github.com/supertokens/supertokens-golang/supertokens"
+)
+
+func MakeBackwardCompatibilityService(recipeInterfaceImpl tplmodels.RecipeInterface, appInfo supertokens.NormalisedAppinfo, sendEmailVerificationEmail func(user evmodels.User, emailVerificationURLWithToken string, userContext supertokens.UserContext), sendPasswordlessLoginEmail func(email string, userInputCode *string, urlWithLinkCode *string, codeLifetime uint64, preAuthSessionId string, userContext supertokens.UserContext) error) emaildelivery.EmailDeliveryInterface {
```

Comment:
You do not need to pass `recipeInterfaceImpl` since it's not used anywhere in this function. (Please make sure that in other backward compatibility functions in other recipes too you are passing only what's needed)